care/facility/models/testing_lab.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth import get_user_model
+from django.core.validators import RegexValidator
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from phonenumber_field.modelfields import PhoneNumberField
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'phonenumber_field.modelfields'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5348729)